app/lib/clients/weather_client.rb

```diff
@@ -1,5 +1,5 @@
 class Clients::WeatherClient
-  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}'.freeze
+  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}&units=imperial'.freeze
```

Comment:
unclear why we're touching this. Maybe put it in its own commit? I think this is a separate idea.

Reply:
The original WEATHER_API_URL has standard unit which gives the data in m/s. Adding units= imperial gives the data in mph.

Reply:
This should be a separate fix commit.